src/styles/app-button/AppButton.tsx

```diff
@@ -0,0 +1,59 @@
+import { ForwardedRef, forwardRef } from "react";
+
+import classnames from "classnames";
+
+import "@/styles/app-button/AppButton.scss";
+import { AppButtonProps } from "@/styles/app-button/AppButton.types";
+import AppSpinner from "@/styles/app-spinner/AppSpinner";
+import AppTypography from "@/styles/app-typography/AppTypography";
+
+// TODO: add link variant
+const AppButton = forwardRef(function (
+  {
+    variant = "primary",
+    size = "md",
+    fullWidth = false,
+    isLoading = false,
+    disabled = false,
+    inline = true,
```

Comment:
I think this prop is redundant, interface inline={false} is not convenient. I suggest using width: "full" | "fit" (fit but default). What do you think about it?

(The position of button will be controlled by outer divs)

Reply:
We shouldn't leave redundant props. There's no inline buttons in our app. I can replace `fullWidth` prop with suggested one